cerberus/aws_auth.py

```diff
@@ -68,7 +69,7 @@ def get_token(self):
         token = resp.json()['client_token']
         iam_principal_arn = resp.json()['metadata']['aws_iam_principal_arn']
         if self.verbose:
-            print('Successfully authenticated with Cerberus as {}'.format(iam_principal_arn), file=sys.stderr)
+            logger.debug('Successfully authenticated with Cerberus as {}'.format(iam_principal_arn), file=sys.stderr)
```

Comment:
We want this to be info logging please, we spend a lot of time on slack supporting this client and would rather have it be noisy than deal with telling customers to enable debug logging over and over.

Reply:
- [ ] make this be logged by default